### PR TITLE
docs: OSS delete user and cleanup placeholders (closes #514)

### LINKED ISSUE #514: api-reference: Users - delete a user

### PR DESCRIPTION
**Pull request by jstirnaman**
- Add descriptions for OSS DELETE /api/v2/users/USER
- Cleanup placeholders and fix text in permissions.
- Add missing errors.

### CODE REVIEW

File: src/common/paths/me_password.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ put:
 
     #### InfluxDB Cloud
 
-    - Doesn't allow you to manage passwords through the API.
+    - Doesn't allow you to manage user passwords through the API.
```
Comment (kelseiv) [inline src/common/paths/me_password.yml:21]:
```suggestion
    - Doesn't let you manage user passwords through the API.
```

File: src/common/paths/me_password.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ put:
 
         #### InfluxDB Cloud
 
-        - Doesn't allow you to manage through the API; always responds with this status.
+        - Doesn't allow you to manage user passwords through the API; always responds with this status.
```
Comment (kelseiv) [inline src/common/paths/me_password.yml:68]:
```suggestion
        - Doesn't let you manage user passwords through the API; always responds with this status.
```

**Review by kelseiv — APPROVED** [review]

**Comment by kelseiv** [comment]
minor edits

File: src/common/paths/orgs_orgID_members.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ get:
 
     - `read-orgs INFLUX_ORG_ID`
 
-    `INFLUX_ORG_ID` is the ID of the organization that you retrieve a list of
+    _`INFLUX_ORG_ID`_ is the ID of the organization that you retrieve a list of
     members from.
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID_members.yml:31]:
```suggestion
    _`INFLUX_ORG_ID`_ is the ID of the organization that you want to retrieve a list of
    members for.
```

File: contracts/cloud.json [hunk 4]
```diff
@@ -1614,7 +1614,7 @@
...
...that the write or delete is queued.\n  To ensure that InfluxDB Cloud handles writes and deletes in the order you request them,\n  wait for a response before you send the next request.\n\n  Because writes are asynchronous, data might not yet be written\n  when you receive the response.\n\n#### InfluxDB OSS\n\n- Validates the request, handles the write synchronously,\n  and then responds with success or failure.\n- If all points were written successfully, responds with HTTP `204` status code;\n  otherwise, returns the first line that failed.\n\n#### Required permissions\n\n- `write-buckets` or `write-bucket BUCKET_ID`.\n\n  _`BUCKET_ID`_ is the ID of the destination bucket.\n\n#### Rate limits (with InfluxDB Cloud)\n\n`write` rate limits apply.\nFor more information, see [limits and adjustable quotas](https://docs.influxdata.com/influxdb/cloud/account-management/limits/).\n\n#### Related guides\n\n- [Write data with the InfluxDB API]({{% INFLUXDB_DOCS_URL %}}/write-data/developer-tools/api)\n- [Optimize writes to InfluxDB]({{% INFLUXDB_DOCS_URL %}}/write-data/best-practices/optimize-writes/)\n- [Troubleshoot issues writing data]({{% INFLUXDB_DOCS_URL %}}/write-data/troubleshoot/)\n",
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply contracts/cloud.json:1617]: > looks like we tell them twice about the status code--or are we sending two status codes--one for queued and one for write?:
> 
> 2. If the write is queued, responds with an HTTP `204` status code.
> 3. Handles the write asynchronously and reaches eventual consistency. An HTTP `2xx` status code acknowledges that the write or delete is queued.

There's a paragraph break there before the explanation ("Handles..."), but I'll look it over in the UI and see if we can improve.

**Pull request merged**